themes-default/slim/views/vue-components/root-dirs.mako

```diff
@@ -0,0 +1,146 @@
+<style>
+    .root-dirs-selectbox,
+    .root-dirs-selectbox select,
+    .root-dirs-controls {
+        width: 100%;
+        max-width: 430px;
+    }
+
+    .root-dirs-selectbox {
+        padding: 0 0 5px;
+    }
+
+    .root-dirs-controls {
+        text-align: center;
+    }
+</style>
+<script type="text/x-template" id="root-dirs-template">
+    <div id="root-dirs-wrapper">
+        <div class="root-dirs-selectbox">
+            ## @TODO: remove id attribute (use Vue $refs to access component data)
+            <select v-model="selectedDir" name="rootDir" id="rootDirs" size="6">
+                <option v-for="(curDir, index) in rootDirs" :value.number="index">{{markDefault(index) + curDir}}</option>
+            </select>
```

Comment:
This setup is now causing me issues:
- If I use `v-model="selectedDir"` and set `:value.number="index"`, I need to add an attribute for the raw root dir path - `:data-path="curDir"`, but that means forms that use [`VueSubmitForm`](https://github.com/pymedusa/Medusa/blob/9f4e60a05106bbd6e27573c778c31f299f1568c3/themes-default/slim/static/js/vue-submit-form.js) take the value, which is the index, and not the actual path.
- If I use `v-model="selectedDir"` and set `:value="curDir"` I can't update the selection via Vue?

Am I missing something big here?

Reply:
Have you tried without value being set? `v-model` updates value so just by having the option set as selected it should fine.

Reply:
That's exactly my point. `v-model` only works with one value. I don't remember if I tried not setting a value **with that `v-for` in place. I assume it's going to use the first variable as the value, which is the path.
That means I gotta change everything because then I can't set selected/default via the option index.

I assume I have to use something new for that - is it this?
```
computed: {
    selectedDir: {
        get () {
        },
        set (newValue) {
        }
    }
}
```

Reply:
Oh likely. You should be able that without an issue.